examples/domain_templates/gan.py

```diff
@@ -107,17 +107,22 @@ def training_step(self, batch, batch_nb, optimizer_i):
             self.generated_imgs = self.forward(z)
 
             # log sampled images
-            sample_imgs = self.generated_imgs[:6]
-            grid = torchvision.utils.make_grid(sample_imgs)
-            self.logger.experiment.add_image('generated_images', grid, 0)
+            # sample_imgs = self.generated_imgs[:6]
+            # grid = torchvision.utils.make_grid(sample_imgs)
+            # self.logger.experiment.add_image('generated_images', grid, 0)
```

Comment:
why do we need to remove image generation? that's important to show

Reply:
I just move it in the callback `on_epoch_end()` since I personally feel it is more resonable to show generated images after each epoch rather than after every batch. In the begining when the generator is less trained, the generated images make little sense so we may not need to show them in every batch. 

BTW some hidden callbacks like `on_epoch_end` are like Easter eggs, have you considered pointing them out as optionals in the README? :-)